src/utils/self_test/self_test.c

```diff
@@ -1652,20 +1699,217 @@ int parse_message_sizes_string(const char *pch,
 	return 0;
 }
 
+/*
+ *********************************
+ * START: Add libconfigini macros
+ *********************************
+ */
+
+#define LOG_ERR(fmt, ...)	\
+	fprintf(stderr, "[ERROR] <%s:%d> : " fmt "\n",\
+		__func__, __LINE__, __VA_ARGS__)
+
+#define LOG_INFO(fmt, ...)	\
+	fprintf(stderr, "[INFO] : " fmt "\n", __VA_ARGS__)
+
+
+#define CONFIGREADFILE		"../etc/config.cnf"
+#define CONFIGSAVEFILE		"../etc/new-config.cnf"
+
+#define ENTER_TEST_FUNC							\
+	do {								\
+		LOG_INFO("%s", "\n-----------------------------------------");\
+		LOG_INFO("<TEST: %s>\n", __FUNCTION__);			\
```

Comment:
(style)  __func__ should be used instead of gcc specific __FUNCTION__

Reply:
fixed